org_status/org_hosts/github.py

```diff
@@ -14,28 +14,29 @@ class GitHubOrg(OrgHost):
     HostName = 'github'
     StatusProvider = [TravisBuildStatus, AppVeyorStatus]
 
-    def __init__(self, token, group, **kargs):
+    def __init__(self, token, group, fetch, **kargs):
```

Comment:
Better call this parameter as `sync`, set it to default to `True` and add this to the parent class definition and make it a read only prop.